### PR TITLE
Translate to Brazilian Portuguese the modules section of the basic training

### PR DESCRIPTION
**Pull request by iaradsouza1**
## Description

This is a first version for modules translation to brazilian Portuguese (https://github.com/nextflow-io/training/issues/190).

### CODE REVIEW

File: docs/basic_training/modules.pt.md [hunk 1]
```diff
@@ -0,0 +1,472 @@
...
+        WORLD!
+        HELLO
+        WORLD!
+        HELLO
+        ```
+
+!!! tip
+
+    Você pode armazenar cada processo em arquivos separados em subpastas separadas ou combinados em um arquivo grande (ambos são válidos).
+    Você pode encontrar exemplos disso em repositórios públicos, como o [tutorial Seqera RNA-Seq](https://github.com/seqeralabs/rnaseq-nf/tree/master/modules) ou em pipelines do nf-core, como [nf-core/rnaseq](https://github.com/nf-core/rnaseq/tree/master/modules/nf-core/modules).
```
Comment (mribeirodantas) [inline docs/basic_training/modules.pt.md:152]:
```suggestion
    Você pode encontrar exemplos disso em repositórios públicos, como no [tutorial de RNA-Seq da Seqera](https://github.com/seqeralabs/rnaseq-nf/tree/master/modules) ou em pipelines do nf-core, como o [nf-core/rnaseq](https://github.com/nf-core/rnaseq/tree/master/modules/nf-core/modules).
```

File: docs/basic_training/modules.pt.md [hunk 2]
```diff
@@ -0,0 +1,472 @@
...
+    my_data = CONVERTTOUPPER.out.upper
+}
+
+workflow {
+    my_pipeline(Channel.of(params.greeting))
+    my_pipeline.out.my_data.view()
+}
+```
+
+O resultado do snippet acima pode ser acessado usando `my_pipeline.out.my_data`.
```
Comment (mribeirodantas) [inline docs/basic_training/modules.pt.md:366]:
```suggestion
O resultado do trecho de código acima pode ser acessado usando `my_pipeline.out.my_data`.
```

File: docs/basic_training/modules.pt.md [hunk 3]
```diff
@@ -0,0 +1,472 @@
...
+
+A execução do script principal acima deve imprimir:
+
+```console
+Olá mundo!
+```
+
+## Notas de migração DSL2
+
+Para visualizar um resumo das alterações introduzidas quando o Nextflow migrou de DSL1 para DSL2, consulte as [notas de migração DSL2](https://www.nextflow.io/docs/latest/dsl2.html#dsl2-migration-notes) na documentação oficial do Nextflow.
```
Comment (mribeirodantas) [inline docs/basic_training/modules.pt.md:472]:
```suggestion
Para visualizar um resumo das alterações introduzidas quando o Nextflow migrou da DSL1 para a DSL2, consulte as [notas de migração da DSL2](https://www.nextflow.io/docs/latest/dsl2.html#dsl2-migration-notes) na documentação oficial do Nextflow.
```

**Review by mribeirodantas — CHANGES_REQUESTED** [review]

**Review by iaradsouza1 — COMMENTED** [review]
Thanks for reviewing, @mribeirodantas ! I think it's ok now.

**Review by mribeirodantas — APPROVED** [review]

**Pull request merged**